docs/src/pages/components/drawers/drawers.md

```diff
@@ -49,6 +49,16 @@ const iOS =
 <SwipeableDrawer disableBackdropTransition={!iOS} disableDiscovery={iOS} />;
 ```
 
+### Drawer Discovery
+
+You can make the drawer appear slightly when the user touches the screen near where the drawer is located.
+
+The amount the drawer appears by can be configured using `discoveryAmount`, and it can be disabled using `disableDiscovery`.
+
+If you are on mobile, you can click near the edge of the screen to discover the drawers.
+
+{{"demo": "pages/components/drawers/SwipeableDiscoveryAmount.js"}}
+
```

Comment:
Can you add a note that the demo only works on the mobile viewport? Also, I try to play this feature in iOS chrome but it does not work (Swipe on the left seems to navigate to previous page instead).